src/Relude/Extra/Enum.hs

```diff
@@ -3,12 +3,15 @@
 
 module Relude.Extra.Enum
        ( universe
+       , inverseMap
        , next
        , safeToEnum
        ) where
 
 import Relude
 
+import qualified Data.HashMap.Strict as HM
```

Comment:
Let use `Data.Map.Strict` instead of `HashMap`. `Ord` constraint is more common than `Hashable`. In our cases it's more important for function to be convenient. Performance difference is not significant.